mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/blob/BlobStoreDeletedMessageVault.java

```diff
@@ -54,14 +55,26 @@
 public class BlobStoreDeletedMessageVault implements DeletedMessageVault {
     private static final Logger LOGGER = LoggerFactory.getLogger(BlobStoreDeletedMessageVault.class);
 
+    private static final String BLOBSTORE_DELETED_MESSAGE_VAULT_METRIC = "deletedMessageVault:blobStore:";
+    static final String APPEND_METRIC_NAME = BLOBSTORE_DELETED_MESSAGE_VAULT_METRIC + "append";
+    static final String LOAD_MIME_MESSAGE_METRIC_NAME = BLOBSTORE_DELETED_MESSAGE_VAULT_METRIC + "loadMimeMessage";
+    static final String SEARCH_METRIC_NAME = BLOBSTORE_DELETED_MESSAGE_VAULT_METRIC + "search";
+    static final String DELETE_METRIC_NAME = BLOBSTORE_DELETED_MESSAGE_VAULT_METRIC + "delete";
+    static final String RETENTION_METRIC_NAME = BLOBSTORE_DELETED_MESSAGE_VAULT_METRIC + "retention";
```

Comment:
I'm still not ok with that `retention` wording (I thought I said it during the related code review). Retention is about keeping things. Here the action is basically to drop everything that we should not retain.
I'd like to not propagate that wording mistake too far so we should refactor now.
What about `clearExpiredMessages`?

Reply:
okay! I might missed what you said in the past.

Reply:
IMO we have time to do the name update refactoring here.

Reply:
I renamed it to `deletedExpiredMessages` to make it fit with the method name